src/components/Homepage/homepage.scss

```diff
@@ -124,21 +137,24 @@
   fill: $carbon--blue-40;
 }
 
-// Dots background pattern
+/* Homepage Nav Boxes */
 
-.homepage--dots {
-  position: absolute;
-  z-index: 1;
-  width: 100%;
-  height: 100%;
-  background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 16 16' style='enable-background:new 0 0 16 16;' xml:space='preserve'%3e%3crect width='16' fill='none' height='16'/%3e%3crect x='0' y='0' fill='white' width='1' height='1'/%3e%3c/svg%3e");
-  background-size: 16px;
-  opacity: 0.25;
-}
+.homepage--nav {
+  padding-top: 0rem;
+  padding-bottom: 1px;
+  padding-left: 0 !important;
+  padding-right: 0 !important;
 
-// Needed for dots to appear below content
+  @include carbon--breakpoint('md') {
+    padding: 1px !important;
+  }
+}
 
-.container--homepage .ibm--row {
-  position: relative;
-  z-index: 2;
+.homepage--nav-tile {
+  background-color: black;
```

Comment:
you've been using tokens for other colors, not sure if you meant to use a token here too?